lib/tool_belt/commands/changelog_command.rb

```diff
@@ -21,7 +22,7 @@ def execute
           config.options.namespace,
           systools
         )
-        ToolBelt::Changelog.new(config.options, release_environment)
+        ToolBelt::Changelog.new(config.options, release_environment, short?)
```

Comment:
Why not use `omit_gh_link?`? It's much more descriptive as a variable and a cli option.